src/README.md

```diff
@@ -9,7 +9,122 @@ To build the docs locally, follow the steps mentioned below:
  6. ``cd ../src``
  7. ``make html``
 
+
+ Conda-forge adheres to the NumFOCUS Code of Conduct. (https://numfocus.org/code-of-conduct)
+
+ ## Scope
+
+ This code of conduct applies to all spaces managed by conda-forge, including all public and private mailing lists, issue trackers, wikis, forums, and any other communication channel used by our community. The code of conduct equally applies at conda-forge events and governs standards of behavior for attendees, speakers, volunteers, booth staff, and event sponsors.
+
+
```

Comment:
There should only be one blank line in-between sections in this document, instead of two or more, so I'd recommend going through and taking those extra blank lines out.